### ISSUE TITLE
Podman integration

### ISSUE THREAD

**Issue by mirkobrombin** [mirkobrombin/Atoms#7]
Atoms was born to manage chroots but optional podman (and docker) support is absolutely feasible by exposing the right permissions to Flatpak.

**Comment by 89luca89** [comment]
You can use `flatpak-spawn` or `host-spawn` to interact with host's podman?

**Comment by mirkobrombin** [comment]
Yes https://github.com/mirkobrombin/Atoms/blob/main/atoms/backend/wrappers/podman.py

**Comment by axtloss** [comment]
It would probably be better to use podman-host, the same way the vscode flatpak does it, that way atoms could also benefit from the flatpak extension that vscode uses as it, making podman support optional.

**Comment by mirkobrombin** [comment]
I was watching it and it looks like it installs Podman, so I assume it will be accessed from inside the flatpak itself but spawning it in the host:

https://github.com/flathub/com.visualstudio.code.tool.podman/blob/4cdc1ca5b42880075eea826137d9cbb5890996f3/com.visualstudio.code.tool.podman.yml#L25

**Comment by axtloss** [comment]
Yeah podman-host is just a tool to communicate to the podman in the host, it doesn't actually run podman in flatpak
